docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/hpc-launching-architecture.rst

```diff
@@ -0,0 +1,370 @@
+         with GPUs have properly configured GRES indicating the presence of any GPUs. The default is
+         true. When false, Determined will request ``slots_per_trial`` nodes and utilize only GPU 0
+         on each node unless slurm.slots_per_node is specified. It is the user’s responsibility to
+         ensure that GPUs will be available on nodes selected for the job using other configurations
+         such as targeting a specific resource pool with only GPU nodes or specifying a Slurm
+         constraint in the experiment configuration.
+
+      -  For PBS, the ngpus resource can be used to identify the number of GPUs available on a node.
+
+-  ``default_aux_resource_pool``: The default resource pool to use for tasks that do not need
+   dedicated compute resources, auxiliary, or systems tasks. Defaults to the Slurm/PBS default
+   partition if no resource pool is specified.
+
+-  ``default_compute_resource_pool``: The default resource pool to use for tasks that require
+   compute resources, e.g. GPUs or dedicated CPUs. Defaults to the Slurm/PBS default partition if it
+   has GPU resources and if no resource pool is specified.
+
+-  ``job_project_source``: Identifies the source to be used when generating a Slurm Workload
+   Characterization Key (WCKey), or PBS project name.
+
+Experiment Configuration
+========================
+
+The experiment contributes the following resource scheduling configuration. See the
+:ref:`experiment-config-reference` for the full details for these configuration options.
+
+-  ``resource_pool``: Identifies the queue/partition to be used.
+
+-  ``slots_per_trial``: The number of slots to use for each trial of this experiment. The default
+   value is 1; specifying a value greater than 1 means that multiple GPUs will be used in parallel.
+   Training on multiple GPUs is done using data parallelism.
+
+-  ``slurm``:
+      -  ``sbatch_args``: Array of Slurm options added as ``#SBATCH`` options in the generated batch
+         script
+      -  ``slots_per_node``: The minimum number of slots required for a node to be scheduled during
+         a trial. It must be greater than slots_per_trial. The default value is 1.
+      -  ``gpu_type``: The Slurm gres type of the GPU to be injected into any generated
+         ``--gpus``/``–gres`` expressions. By default, no type is specified.
+
+-  ``pbs``:
+      -  ``pbsbatch_args``: Array of PBS options added as ``#PBS`` options in the generated batch
+         script
+      -  ``slots_per_node``: The minimum number of slots required for a node to be scheduled during
+         a trial. It must be greater than ``slots_per_trial``. The default value is 1.
+
+Slurm Scheduling
+================
+
+All information is passed to Slurm through options in the generated sbatch file. You may identify
+the generated sbatch file and logs using the Slurm command scontrol show job, and then inspect the
```

Comment:
To be clearer, shouldn't it be "You may identify the location of the generated batch file ..."?